travis-scripts/valgrind.sh

```diff
@@ -144,11 +144,10 @@ function check_masterfiles_and_inputs {
 /var/cfengine/bin/cf-agent --version
 
 VG_OPTS="--leak-check=full --track-origins=yes --error-exitcode=1"
-BOOTSTRAP_IP="$(ifconfig | grep -A1 Ethernet | sed '2!d;s/.*addr:\([0-9.]*\).*/\1/')"
 
 valgrind $VG_OPTS /var/cfengine/bin/cf-key 2>&1 | tee cf-key.txt
 check_output cf-key.txt
-valgrind $VG_OPTS /var/cfengine/bin/cf-agent -B $BOOTSTRAP_IP 2>&1 | tee bootstrap.txt
+valgrind $VG_OPTS /var/cfengine/bin/cf-agent -B $HOSTNAME 2>&1 | tee bootstrap.txt
 check_output bootstrap.txt
 
 echo "127.0.0.1" > /var/cfengine/policy_server.dat
```

Comment:
```suggestion
valgrind $VG_OPTS /var/cfengine/bin/cf-agent -B 127.0.0.1 2>&1 | tee bootstrap.txt
check_output bootstrap.txt
```

Reply:
@larsewi we can bootstrap to localhost now, could you make this change in a followup PR?

Reply:
Hmm, wouldn't that bypass much of the networking code?

Reply:
What is the advantage of using localhost?

Reply:
You don't have to bother about the hostname of the host. But if `$(HOSTNAME)` just works, we can just use that, IMHO.

Reply:
127.0.0.1 "always works" and makes the default ACL not allow connections from other machines.

We are already doing this by overwriting `policy_server.dat`, just bootstrapping to 127.0.0.1 is one step less.

Reply:
Sure, my only concern is if bootstrapping to `127.0.0.1` goes through the same code paths as normal bootstrap to a (local) hostname. The valgrind test should have the best possible coverage, after all.